torchvision/prototype/transforms/_geometry.py

```diff
@@ -171,6 +180,26 @@ def forward(self, *inputs: Any) -> Any:
 
 
 class TenCrop(Transform):
+    """
+    Example:
```

Comment:
I would just point to the `FiveCrop` example here to avoid duplicating the example.